test/common/testlib.py

```diff
@@ -519,6 +519,23 @@ def click_action_btn(self, sel, entry=None):
         else:
             self.click(sel + ' button:first-child')
 
+    def try_login(self, user, password, focus_element='login-user-input', authorized=True):
+        """
+        Fills in the login dialog and clicks the button, without
+        expecting any particular result.
+        """
+        self.wait_visible("#login")
+        self.wait_visible("#login-user-input")
+        self.wait_not_visible("#unsupported-browser")
+        self.wait_not_present("#login-button:disabled")
+        # :focus selector doesn't work in phantomjs
+        if focus_element is not None:
+            self.wait_js_cond("document.activeElement.getAttribute('id') === '{}'".format(focus_element))
```

Comment:
(copying from #13473): We don't need all this any more. I know it was copied from check-login, but by now our testlib.py has caught up to make this much simpler. We can continue to do what the regular `login_and_go()` does.

phantomjs hasn't been a thing for more than two years, and these days functions like set_val() and click() all wait by themselves for the element to become present, visible, and clickable. The only line which isn't completely redundant is `self.wait_not_visible("#unsupported-browser")`. If you want, you can keep that, although I think that should only be part of a specialized test for the unsupported browser page functionality, and not necessarily happen on every test. But I'm fairly indifferent about that one.